packages/utils/src/misc.ts

```diff
@@ -395,3 +396,17 @@ export async function handleDeeplinkRedirect({
     console.error(err);
   }
 }
+
+export async function getDeepLink(store: IKeyValueStorage, key: string) {
+  try {
+    const deepLink = await store.getItem(key);
+    if (deepLink) return deepLink;
+
+    // check localStorage as fallback
```

Comment:
LGTM code-wise, but not 100% clear to me why we need this behaviour? Assumption being that `store` now refers to IndexedDB by default and we want to ensure we still check localStorage too in a browser env?

Reply:
yes, some modal versions set the deeplink in `localStorage` directly and are not using the `Core's storage` so this fallbacks covers them